docs/configuration.md

```diff
@@ -1062,7 +1062,7 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.ui.showConsoleProgress</code></td>
-  <td>true</td>
+  <td>false</td>
   <td>
     Show the progress bar in the console. The progress bar shows the progress of stages
```

Comment:
Shall we mention that this is enabled in shells by default?

Reply:
+1 for @HyukjinKwon 's comment.

Reply:
@HyukjinKwon @dongjoon-hyun I will add a mention about this.